src/OpenLoco/src/Localisation/StringIds.h

```diff
@@ -1648,9 +1648,12 @@ namespace OpenLoco::StringIds
     constexpr string_id buffer_2040 = 2040;
 
     constexpr string_id buffer_2042 = 2042;
-
+    constexpr string_id not_enough_space_for_graphics = 2043;
+    constexpr string_id too_many_objects_of_this_type_selected = 2044;
     constexpr string_id buffer_2045 = 2045;
```

Comment:
technically these aren't really buffers (2045 and 2042) they are something else and comprise of an initial string followed by a buffer section.